### DISCUSSION TITLE
Error while importing CSV in noco DB [Sep 24]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#6534] [Discord Threads]
> 🙋‍♂️ **iqbalpb**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### can you share your project info?  
#  
> 🙋‍♂️ **iqbalpb**  
#### Can I send you a direct message containing the details?   
#### cc: <@495466997304262658>  
#  
> 👨‍🚀 **wingkwong**  
#### sure. please DM me.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1144491878423724032)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
